packages/help-center/src/components/help-center-header.scss

```diff
@@ -37,12 +37,12 @@
 
```

Comment:
There is a focus state on the header now that it is all a button.

<img width="476" alt="Screen Shot 2024-10-03 at 16 00 46" src="https://github.com/user-attachments/assets/cc17faec-51e7-4b4c-ac12-bc6bfb884074">

Reply:
It looks like the button is overflowing out of the container and causing it to be draggable outside the container.


https://github.com/user-attachments/assets/4d02af0a-57e7-4fec-9342-4181b6364fa6

Reply:
Something is making the border-radius `7px` when it should be 2 all around.

<img width="430" alt="Screen Shot 2024-10-03 at 16 43 42" src="https://github.com/user-attachments/assets/ea992bef-66ca-4aae-ba31-6caf86f89cfa">